packages/azure-http-specs/specs/azure/client-generator-core/client-initialization/client.tsp

```diff
@@ -199,3 +199,48 @@ interface ParamAlias {
   withAliasedName is Service.ParamAlias.withAliasedName;
   withOriginalName is Service.ParamAlias.withOriginalName;
 }
+
+@global.Azure.ClientGenerator.Core.client({
+  name: "ParentClient",
+  service: Service,
+})
+namespace ParentClient {
+  @scenarioDoc("""
+    Client for testing a path parameter (blobName) moved to client level, in child client.
+    
+    The child client can be initialized individually, or via its parent client.
+    
+    Parameters elevated to client level:
+    - blobName: "sample-blob" (path parameter)
+    
+    Expected client usage:
+    ```ts
+    // via ParentClient
+    const client = new ParentClient.getChildClient({
+      blobName: "sample-blob"
+    });
+    
+    // directly
+    const client = new ChildChild({
```

Comment:
wondering the reason for the name `ChildChild`, given the interface name used below is `ChildClient`

Reply:
Thanks, fixed the typo.